packages/api/helper/getRawNominationDataFromGoogleSheets.js

```diff
@@ -4,7 +4,7 @@ const rangePar = 'Sheet1';
 const client = new google.auth.JWT(
   process.env.GOOGLE_SHEETS_CLIENT_EMAIL,
   null,
-  process.env.GOOGLE_SHEETS_PRIVATE_KEY,
+  // process.env.GOOGLE_SHEETS_PRIVATE_KEY,
```

Comment:
Should this change be discarded?

Reply:
That is a great question, I am not certain if that is needed for production or just commented for any reason.